docs/user/ovirt/install_ipi.md

```diff
@@ -46,6 +46,15 @@ DNS and LB services but is a platform provider. See also [OpenShift-Metal³ kni-
     provider - see this ansible role https://github.com/oVirt/ovirt-ansible-image-template
     Use it to make a template from the glance image.
 
+3. Create the ~/.ovirt/ovirt-config.yaml file with contents similar to this.
```

Comment:
This should not be required, as the installer will prompt you fir credentials and store if not already present.

Reply:
there was a bug with storing but it should be fixed with https://github.com/openshift/installer/pull/3135